tests/test_redis.py

```diff
@@ -253,3 +254,32 @@ def testCheckTimeStampInvalid(self):
         result = p.check_time_stamps(tenantid, serverid, p.range(serverid, tenantid), p5)
 
         self.assertEqual(expected, result)
+
+    def testILikethegirlwhoisnext2me(self):
```

Comment:
Apart that the name of the test I assume that it is not correct, you have two methods with the same name...